buildSrc/src/main/java/org/elasticsearch/gradle/precommit/TestingConventionsTasks.java

```diff
@@ -295,7 +288,10 @@ private boolean seemsLikeATest(Class<?> clazz) {
             }
             return false;
         } catch (NoClassDefFoundError e) {
-            throw new IllegalStateException("Failed to inspect class " + clazz.getName(), e);
+            // Include the message to get more info to get more a more useful message when running Gradle without -s
+            throw new IllegalStateException(
+                "Failed to inspect class " + clazz.getName() + ". Missing class? " + e.getMessage(),
```

Comment:
The cause is already attached, is duplicating the cause's message really necessary?

Reply:
I did it so we get the name of the missing class even when the build does not run with  `-s`. Would be useful in CI.